magento/pwa-studio
# Cypress test results reporting

## Description

This PR adds the ability for Cypress to report its test results in the form of JSON data which danger CLI can use to report on the PR.

Should be merged along with https://github.com/magento-commerce/pwa-studio-cicd/pull/94

## Related Issue

Closes PWA-2119

### Verification Stakeholders

@dpatil-magento 
@sharkySharks 

## Verification Steps

Since this is a CICD PR, replicate this change in the comops repo and use the Danger CLI branch mentioned in the [CICD PR](https://github.com/magento-commerce/pwa-studio-cicd/pull/94) for the cypress codebulid job. You should see Cypress failures (if any) showing up in the danger comment.
